File: nivel01/respostas/c/exercicio04.c
#include <stdio.h>

int main() {
	
	int dias, meses, anos, total;

	printf("Digite a idade em anos: ");
	scanf("%d", &anos);

	printf("Digite os meses: ");
	scanf("%d", &meses);

	printf("Digite os dias: ");
	scanf("%d", &dias);

	total = anos * 365 + meses * 30 + dias;

	printf("Idade total em dias: %d", total);
	printf("\n");

	return 0;

}

File: nivel01/respostas/ruby/exercicio04.rb
print "Idade em anos: "
anos = gets.to_i
print "Idade em meses: "
meses = gets.to_i
print "Idade em dias: "
dias = gets.to_i

total = (anos * 360) + (meses * 30) + dias

puts "Sua idade em dias é : ", total
